src/autopas/containers/octree/OctreeLeafNode.h

```diff
@@ -79,7 +79,18 @@ class OctreeLeafNode : public OctreeNodeInterface<Particle>, public FullParticle
     }
 
     if ((this->_particles.size() < this->_treeSplitThreshold) or anyNewDimSmallerThanMinSize) {
+      // sanity check that ensures that only particles of the cells OwnershipState can be added. Note: is a cell is a
```

Comment:
```suggestion
      // sanity check that ensures that only particles of the cells OwnershipState can be added. Note: if a cell is a
```

Reply:
done in [37e2621](https://github.com/AutoPas/AutoPas/pull/782/commits/37e26218410380f7bc586d66bd83b316d23a3d28)